medusa/helpers/__init__.py

```diff
@@ -1797,6 +1797,7 @@ def get_broken_providers():
         return []
 
     log.info('Broken providers found: {0}', response)
+    response = [u'sdbits', u'extratorrent', u'freshontv', u'bithdtv']
```

Comment:
oow oeps. Don't forget to remove this line, before merging this.

Reply:
_Friendly reminder_ 😄

Reply:
ok done